### ISSUE TITLE
Problem on querying ES for Battle Chess

### ISSUE THREAD

**Issue by DenisAraujo68** [Ensepro/ensepro-core#15]
The weird situation below was detected:
```
# ensepro -resposta -tr -frase "Há algum jogo eletrônico chamado Battle Chess?"
Analisando frase(s)... done
-> Frase 1: Há algum jogo eletrônico chamado Battle Chess?
--> Tipo: TipoFrase={tipo=algum, confianca=0.89, ids=[4]}
--> Voz: Voz.ATIVA
--> Termos Relevantes:
----> TR 0: [H:n|5| jogo]
----> TR 1: [DN:adj|6| eletrônico]
----> TR 2: [DN:adj|7| chamado]
----> TR 3: [DN:prop|8| Battle_Chess]
Nenhuma resposta encontrada.
```
Log file: [ensepro.log](https://github.com/Ensepro/core/files/2469718/ensepro.log)

**Comment by alencarrh** [comment]
@DenisAraujo68 I didn't really understand what did you mean with **_weird_**, but I tested it here, and it working fine.  

Can you test it again and see if the problem still happens?

**Comment by DenisAraujo68** [comment]
@alencarrh, the ENSEPRO system are saying that there is no one triple with the word "jogo", for example, what is hard to believe.
But I tested the system and now I understood what happened. There are 2 TRs in the question: *jogo* and *Battle Chess*.  There was a bug that was removing the word "jogo" from the list of TRs  (issue #13). 
So *battle_chess* was the uniq TR identified by the system and there are no triples containing this string.
This issue can be closed, because is was resolved in #13

**Comment by DenisAraujo68** [comment]
@alencarrh, the ENSEPRO system are saying that there is no one triple with the word "jogo", for example, what is hard to believe.
But I tested the system and now I understood what happened. There are 2 TRs in the question: jogo and Battle Chess. There was a bug that was removing the word "jogo" from the list of TRs (issue #13).
So battle_chess was the unique TR identified by the system and there are no triples containing this string.
I'm closing this issue because it is was resolved in #13
